src/Backend/Modules/Pages/Resources/config/doctrine.yml

```diff
@@ -0,0 +1,11 @@
+doctrine:
+    dbal:
+        types: ~
+    orm:
+        mappings:
+            module_extra:
+                type: annotation
+                is_bundle: false
+                dir: "%kernel.project_dir%/src/Backend/Modules/Pages/Domain/ModuleExtra"
+                alias: ModuleExtra
```

Comment:
Wouldn't it be a better idea to use ModuleEntity as the alias? So, in this case, it would be PagesModuleExtra, that way if a different module registers a ModuleExtra it won't clash

Reply:
It's a good idea. Changed in c13b0479